client/src/index.js

```diff
@@ -17,7 +26,16 @@ const store = createStoreWithMiddleware(combinedReducer);
 ReactDOM.render(
   <Provider store={store}>
     <Router>
-      <App />
+      <div>
```

Comment:
The routes have been moved to the same level as the router, in `index.js`, obsoleting the App.js file. Each of the wireframe pages is now skeletonize and ready for further development. They will all be reachable from within the app with the buildout of a Navbar.